src/routers/peers.py

```
@@ -39,10 +39,10 @@
 )
 async def get_peers(
     workspace_id: str = Path(..., description="ID of the workspace"),
-    options: Optional[schemas.PeerGet] = Body(
+    options: schemas.PeerGet | None = Body(
         None, description="Filtering options for the peers list"
     ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)